packages/block-library/src/table/index.js

```diff
@@ -89,6 +89,11 @@ export const settings = {
 		foot: getTableSectionAttributeSchema( 'foot' ),
 	},
 
+	styles: [
+		{ name: 'default', label: __( 'Regular' ), isDefault: true },
+		{ name: 'striped', label: __( 'Striped' ) },
```

Comment:
I'm open to any verbiage, but definitely good to change now rather than later.

I actually intended to call this "zebra striped", but just striped seemed better based on a Google search.

This reminds me, I meant to ask @youknowriad: can we collapse the alignments for the table block by default no matter the breakpoint?

Reply:
Unfortunately, with the way they're defined in this block (hook triggered by `supports: { align: true }`) it's not possible to do so without impact on all the other blocks using this hook. I think I'm fine duplicating the alignment hook in the block (I don't like this support key) but I know @jorgefilipecosta was doing some work on that, so he might know more.

Reply:
"Stripes" looks good to me.